modules/odc-creating-sample-applications.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-creating-sample-applications.adoc
+
+[id="odc-creating-sample-applications_{context}"]
+= Creating Sample applications
+
+You can use the basic sample applications in the *+Add* flow of the *Developer* perspective to create, build, and deploy applications quickly.
+
+The following procedure explains the *Samples* option in the *Developer* perspective to create a sample application.
+
+.Procedure
+
+. In the *+Add* view, click *Samples* to see the *Samples* page. 
+. On the *Samples* page, select one of the available sample applications to see the *Create Sample Application* form. 
+. In the *Create Sample Application Form*:
+* In the *Name* field, the deployment name is displayed by default. You can modify this name as required.
+* In the *Builder Image Version*, a builder image is selected by default. You can modify this image version by using the *Builder Image Version* drop-down list.
+* A sample Git repository URL is added by default.
+. Click *Create* to create the sample application. The build status of the sample application is displayed on the *Topology* view. After the sample application is created, you can see the deployment added to the application.
```

Comment:
One suggestion (take it or leave it :smiley_cat:):  
. Click *Create* to create the sample application. 
+
The build status of the sample application is displayed on the *Topology* view. After the sample application is created, you can see the deployment added to the application.

Reply:
Hi @bmcelvee 

Thanks for the suggestion! However in the docs, **Topology** is a UI Element and we have used **Create** in bolds as well. It is not intuitive enough to use italics for the same. Would love to know your thoughts about the same.

Reply:
Oh how strange! Must have been a formatting quirk from Git - I was just suggesting adding a hard break following the procedural step, and nesting the build status portion underneath. Not a big deal though!